src/utils/amplitude.ts

```diff
@@ -0,0 +1,25 @@
+import * as amplitude from '@amplitude/analytics-browser';
+
+
+ // Define a type for the event properties
+ interface EventProperties {
+   [key: string]: any;
+ }
+
+ export const initAmplitude = (): void => {
+   if (typeof window !== 'undefined') {
+     amplitude.init(process.env.NEXT_PUBLIC_AMPLITUDE_API_KEY as string, {
+       defaultTracking: true,
+     });
```

Comment:
We will need to initialize with the user ID from Highlight here

Reply:
Okay done